src/ProfStef-Core/Lesson.class.st

```diff
@@ -10,8 +10,9 @@ Class {
 		'title',
 		'lesson'
 	],
-	#category : 'ProfStef-Core',
-	#package : 'ProfStef-Core'
+	#category : 'ProfStef-Core-core',
```

Comment:
If the package already is "XXX-Core" then better use "Base" for the tag/category"